v2/boost-sdk/boost-core/create-boost.mdx

```diff
@@ -3,6 +3,143 @@ title: "Create Boosts"
+  incentives,
+});
+```
+
+<Config />
+
+```ts action.ts
+import { core } from './config'
+import { 
+  ActionStep, 
+  SignatureType, 
+  FilterType, 
+  PrimitiveType, 
+  ActionClaimant,
+} from '@boostxyz/sdk/Actions/EventAction'
+import { selectors as funcSelectors } from '@boostxyz/signatures/functions'
+import { Hex, toHex } from 'viem';
+import { zora } from 'viem/chains';
+
+const targetContract = '0x777777722d078c97c6ad07d9f36801e653e356ae'
+const mintSelector = funcSelectors['mint(address mintTo,uint256 quantity,address collection,uint256 tokenId,address mintReferral,string comment)'] as Hex;
+
+const commonParams = {
+  chainid: zora.id,
+  signature: mintSelector,
+  signatureType: SignatureType.FUNC,
+  targetContract: targetContract,
+} as const
+
+// Target a specific collection on Zora TimedSalesStrategy contract
+const mintActionStep1: ActionStep = {
+  ...commonParams,
+  actionParameter: {
+    filterType: FilterType.EQUAL,
+    fieldType: PrimitiveType.ADDRESS,
+    fieldIndex: 2, // collection
+    filterData: '0x3263023c87502f1676f00df902b1237f93da26a9',
+  },
+};
+
+// Target a specific tokenId on the collection
+const mintActionStep2: ActionStep = {
+  ...commonParams,
+  actionParameter: {
+    filterType: FilterType.EQUAL,
+    fieldType: PrimitiveType.UINT,
+    fieldIndex: 3, // tokenId
+    filterData: toHex(1),
+  },
+}
+
+// Target the mintTo field for the claimant
+const actionClaimant: ActionClaimant = {
+  ...commonParams,
+  fieldIndex: 0, // mintTo
+}
+
+export const eventActionPayload = {
+  actionClaimant,
+  actionSteps: [mintActionStep1, mintActionStep2],
+};
+```
+
+```ts incentives.ts
+import { StrategyType } from '@boostxyz/sdk'
+import { parseEther } from 'viem'
+
+export const incentives = [
+  core.ERC20Incentive({
+    asset: "0xf3B2d0E4f2d8F453DBCc278b10e88b20d7f19f8D",
+    reward: parseEther("0.5"),
+    limit: 100n,
+    strategy: StrategyType.POOL,
+    manager: "0xMANAGER_ADDRESS"
+  }),
+],
+```
+</CodeGroup>
+
+#### Parameters
+
+<ParamField path="_boostPayload" type={<a href="https://boost-protocol.vercel.app/interfaces/CreateBoostPayload.html">CreateBoostPayload</a>} required>
+    <Expandable title="properties">
+      <ParamField path="budget" type={<a href="https://boost-protocol.vercel.app/types/Budget.html">Budget</a>} required />
+      <ParamField path="action" type={<a href="https://boost-protocol.vercel.app/types/Action.html">Action</a>} required />
+      <ParamField path="validator" type={<a href="https://boost-protocol.vercel.app/types/Validator.html">Validator</a>} />
+      <ParamField path="allowList" type={<a href="https://boost-protocol.vercel.app/types/AllowList.html">AllowList</a>} />
+      <ParamField path="incentives" type={<a href="https://boost-protocol.vercel.app/types/Incentive.html">Incentive[]</a>} required />
+      <ParamField path="protocolFee" type="bigint" />
+      <ParamField path="maxParticipants" type="bigint" />
```

Comment:
not sure if best place for it is here, but we need somewhere prominent to point out that budgets will transfer a multiple of maxParticipants worth of assets into the incentive, it can be confusing